metron-deployment/amazon-ec2/README.md

```diff
@@ -126,6 +126,10 @@ To provision only subsets of the entire Metron deployment, Ansible tags can be s
 ./run.sh --tags="ec2,sensors"
 ```
 
+### Setting REST API Profile
+
```

Comment:
Can we say something for people like me..  along the lines of
"Spring profiles are used for x,y,z.  By default the dev profile is selected.  Change the values of....  For more information on setting the profiles and their use please see ....

Reply:
Spring profiles are documented in the REST README:  https://github.com/apache/metron/tree/master/metron-interface/metron-rest#spring-profiles.  

Is there something we can do to make the REST README more accessible?  I feel like a lot of questions people ask are already answered there but no one ever reads it.  What can we do to make it more useful?  Table of contents maybe?  I would be happy to take that on in a follow-up PR.

Reply:
That's what I linked to in that README change @merrimanr and @ottobackwards. I didn't want to duplicate the REST docs, but agree with Otto about having a reference there.

Reply:
Just linking to the other doc, without the user knowing even a little of why they need to look at it is not much better.  I think my blurb is appropriate.

Reply:
Reading that back, it seems a little stronger than I intended.  Sorry.  I don't think everyone following these deployment steps is necessarily going to know why they need to start tripping through readme land without some context.  We have a lot of people trying deployment and having problems who are not as expert

Reply:
@ottobackwards I've made another round of improvements after discussing what the profiles do with @merrimanr. Hopefully this addresses your concerns reasonably in the short term. I added this item to the PR in the desire to gain some functional parity with full dev in the short term. But I think we might want to revisit how we setup the REST API, profiles, and user creation in another follow-on PR.

Reply:
Just caught your comment above @ottobackwards. No apology necessary, it was/is a perfectly reasonable request. At first I couldn't provide the doc for it because, as I thought it through, I couldn't give a concise answer. I honestly didn't have a clear picture in my mind of what all happens when setting those values. It was definitely worthwhile to cross-check the docs against the source and understand a bit more fully what those profiles are doing.